books.rb

```diff
@@ -0,0 +1,16 @@
+class Books
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  # setters and getters for instance variables
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] This method suppose to create rentals and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association, So please fix that.
[Hint] the method will be like that:
```ruby
def add_rental(person, date) 
    Rental.new(date, self, person)
end
```